Dockerfile

```diff
@@ -14,12 +13,15 @@ FROM build-setup AS build-dependencies
 
 # Cache gopath dependencies for faster builds
 # Newer projects should opt for go mod vendor for reliability and security
-WORKDIR /app/src
+RUN mkdir /app
+RUN mkdir /app/src
 COPY src/go.mod /app/src
 COPY src/go.sum /app/src
-RUN go mod download
 
 # FIX: This generates code marked by `go:build relic` and `+build relic`. See `combined_verifier_v3.go`.
+# FIX: This is not needed, if vendor/ is used
+WORKDIR /app/src
+RUN go mod download
 RUN go mod download github.com/onflow/flow-go/crypto@v0.24.3

```

Comment:
We may want to add here some documentation why this exact version we use. It is basically the latest stable from flow-go